todo-mvc/src/containers/TodoAppContainer.ts

```diff
@@ -0,0 +1,44 @@
+import { Container } from '@dojo/widget-core/Container';
+import { TodoApp } from './../widgets/TodoApp';
+import uuid from '@dojo/core/uuid';
+import { Store } from 'le-store/store';
+import {
+	addTodoProcessWithPost,
+	editTodoProcess,
+	toggleTodoProcess,
+	deleteTodoProcess,
+	toggleAllTodoProcess,
+	clearCompletedProcess,
+	saveTodoProcess,
+	todoInputProcess,
+	clearFailedProcess
+} from './../processes/todoProcesses';
+
+function getProperties(store: Store, properties: any) {
+	const { get, createExecutor } = store;
```

Comment:
The injector provides access to the `store` for the `Container`, peel the `get` and `createExecutor` functions from the store needed to create the properties required by `TodoApp`